src/practitest_firecracker/parser/gherkin.clj

```diff
@@ -58,6 +58,9 @@
              (:map param)] result]]))
          all-params)))
 
+(defn- examples-section-end? [line]
+  (str/starts-with? (str/lower-case (str/trim line)) "scenario:"))
```

Comment:
new scenario -> indicates "examples" are over? if so, does it also work for the last `scenario`?

Reply:
Yes there's another condition in loop - so examples are over if there's an end of file or there's another scenario starting